src/components/articles.jsx

```diff
@@ -0,0 +1,27 @@
+
+const Articles = ({ articles }) => {
+
+    return (
+        <main>
+            <h2>All Articles</h2>
+
+            <ul className='allArticles'>
+           {articles.map((article) => {
+            return <section>
```

Comment:
If it's the section that you return inside the .map() then that must have a unique key.